setup/setup.sh

```diff
@@ -0,0 +1,217 @@
+#!/bin/bash
+
+ORKA_VM_TOOLS_VERSION="${ORKA_VM_TOOLS_VERSION:-3.5.0}"
+CURRENT_USER="${USER}"
+
+GREEN="\033[0;32m"
+YELLOW="\033[1;33m"
+RED="\033[0;31m"
+NC="\033[0m"
+
+set -euo pipefail
+
+log() {
+    echo -e "${GREEN}[INFO]${NC} $1"
+}
+
+warn() {
+    echo -e "${YELLOW}[WARN]${NC} $1"
+}
+
+error() {
+    echo -e "${RED}[ERROR]${NC} $1" >&2
+    exit 1
+}
+
+# Check if running as root
+if [[ $EUID -eq 0 ]]; then
+    error "This script should not be run as root. Please run as a regular user with sudo privileges."
+fi
+
+# Check if sudo is available
+if ! command -v sudo &> /dev/null; then
+    error "sudo is required but not available"
+fi
+
+log "Starting MacOS Orka VM setup..."
+
+# Enable Screen Sharing
+enable_screen_sharing() {
+    log "Enabling Screen Sharing..."
+    
+    # Enable Screen Sharing service
+    sudo launchctl load -w /System/Library/LaunchDaemons/com.apple.screensharing.plist
+    
+    log "Screen Sharing enabled"
+}
+
+# Enable Remote Login (SSH)
+enable_remote_login() {
+    log "Enabling Remote Login (SSH)..."
+    
+    local CURRENT_USER="$USER"
+    
+    # Add current user to admin group if not already a member
+    sudo dscl . -append /Groups/admin GroupMembership "$CURRENT_USER" 2>/dev/null || true
+    sudo dscl . -append /Groups/wheel GroupMembership "$CURRENT_USER" 2>/dev/null || true
+    
+    # Ensure home directory exists for current user
+    sudo createhomedir -c -u "$CURRENT_USER" 2>/dev/null || true
```

Comment:
Do we need all of this logic? I do not think that we do. An administrator user will have all of this already